### ISSUE TITLE
sqlite3.OperationalError: database or disk is full

### ISSUE THREAD

**Issue by amizeranschi** [ComparativeGenomicsToolkit/cactus#866]
Hi!

I'm running into the error `sqlite3.OperationalError: database or disk is full` when trying to align several genomes. I'm running Cactus through Docker on a single machine with plenty of resources and I'm quite certain that storage isn't the issue.

Through some googling I've come across [this page](https://stackoverflow.com/questions/16685016/sqlite3-operationalerror-database-or-disk-is-full-on-lustre), which mentions a limit on the number of pages in an sqlite database. Is there a way I can check what this limit is for Toil / Cactus and perhaps increase it?

I'm attaching the errors I'm getting below.

[errors.txt](https://github.com/ComparativeGenomicsToolkit/cactus/files/10256221/errors.txt)

**Comment by amizeranschi** [comment]
Forgot to add, this is the command I'm running:

```
docker run -v $(pwd)/data:/data --rm -it quay.io/comparative-genomics-toolkit/cactus:v2.3.0 cactus ./jobStore genomes.txt genomes.hal
```

**Comment by glennhickey** [comment]
Very weird, I think that's the fist time I've seen that one outside a cluster.  Could you please try running with `--disableCaching`?

**Comment by amizeranschi** [comment]
Yes, I saw that option mentioned in another issue and gave it a try. Out of curiosity, what does it do, exactly? And what would be the effect of using it for all my future Cactus jobs?

I can confirm that it did, indeed, fix the previous error and that the pipeline passed that stage. It then ran for several hours and ended up crashing with a different error message: `ERROR: Got an unexpected character paf cigar string: i`.

Attaching the full error messages below. Should I open a new issue for this?

[new_error.txt](https://github.com/ComparativeGenomicsToolkit/cactus/files/10261232/new_error.txt)
